src/app/citrix-handler.ts

```diff
@@ -0,0 +1,23 @@
+import { enumerateValues, HKEY, RegistryValueType } from 'registry-js';
+
+export const getCitrixMediaRedirectionStatus = ():
+  | 'inactive'
```

Comment:
you can define here an enum that would have 'inactive' 'supported' or 'unsupported' values.

Reply:
Makes sense, updated